### ISSUE TITLE
Limit User History

### ISSUE THREAD

**Issue by k-z-m-r** [cse442-spring-2020-offering/cse442-project-purehtml#122]
As a developer, I would like to limit the amount of searches stored for a user so that it doesn't overload our memory.

Functionality:
- once we have the user logged in, we check the length of their current searches after appending the new search.
- this requires the user to make a search. 
- here, if the length is greater than 10, we iterate and remove the head until the length is under 10.
- then, we pass it back to the original json.

Task tests:
- make a new user.
- your history page should be empty.
- make a valid search and then check user history.  repeat 9 more times.
- you should see each number increment up until 10, for each search.
- take note of the 10th artist and search an artist different from that artist.
- note that your 10th artist now becomes what your 9th artist was, and the 1st is your most recent search.
- basically, verify that everything was shifted up 1, but cut off after 10.
- you can do this a few times to verify that it works.

This prevents system memory from being flooded, but we can adjust the limit if needed.
